src/NDS.cpp

```diff
@@ -773,7 +773,8 @@ bool DoSavestate(Savestate* file)
         file->Bool32(&LagFrameFlag);
     }
 
-    // TODO: save KeyInput????
+    if (file->IsAtleastVersion(7, 2))
```

Comment:
you don't really need to add that, we don't have any guarantees for savefile compability anyway

Reply:
I like to preserve backwards compatibility, the feature exists, and this is a very easy way to do that at least until a major update.